Source/swiftlint/Helpers/CompilerArgumentsExtractor.swift

```diff
@@ -1,14 +1,19 @@
 import Foundation
 import SourceKittenFramework
 
+private let escapedSpacePlaceholder = "\u{0}"
+
 struct CompilerArgumentsExtractor {
     static func allCompilerInvocations(compilerLogs: String) -> [String] {
         var compilerInvocations = [String]()
         compilerLogs.enumerateLines { line, _ in
             if let swiftcIndex = line.range(of: "swiftc ")?.upperBound, line.contains(" -module-name ") {
-                let invocation = line[swiftcIndex...]
+                let invocation = line
+                    .replacingOccurrences(of: "\\ ", with: escapedSpacePlaceholder)[swiftcIndex...]
                     .components(separatedBy: " ")
+                    .map { $0.replacingOccurrences(of: escapedSpacePlaceholder, with: " ") }
                     .expandingResponseFiles
+                    .map { $0.replacingOccurrences(of: " ", with: escapedSpacePlaceholder) }
```

Comment:
I'm almost positive there is a more elegant solution to the 3 transformations I added here, but will want to think about how it affects the escaping strategy used in `parseCLIArguments` and `expandingResponseFiles`.